roles/install-nfs/templates/deploy-nfs-client.sh.j2

```diff
@@ -0,0 +1,50 @@
+#! /bin/bash
```

Comment:
let us avoid this script and convert to enable resources.

Reply:
done